### PR TITLE
#980: Fix tablo (closes #980)

### LINKED ISSUE #980: Fix tablo

### PR DESCRIPTION
**Pull request by francescogior**
Closes #980

### CODE REVIEW

**Comment by giogonzo** [comment]
ok we have something now.. the real task here is to discover how react-skinnable is broken with respect to revenge/skinnable then?

File: src/tablo/plugins/columnsReorder/columnsReorderGrid.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default (Grid) =>
     columnsOrder: maybe(list(t.String)),
     onColumnsReorder: maybe(t.Function)
   }, { strict: false })
-  @skinnable(contains(Grid))
+  // @skinnable(contains(Grid))
```
Comment (FrancescoCioria) [inline src/tablo/plugins/columnsReorder/columnsReorderGrid.js]: maybe remove completely instead of commenting? (as you've done in other files)

**Comment by FrancescoCioria** [comment]
@francescogior can you briefly explain me why `react-skinnable` was breaking sorting and reorder?

**Comment by francescogior** [comment]
> @francescogior can you briefly explain me why react-skinnable was breaking sorting and reorder?

I didn't find out yet. @ascariandrea is also investigating

**Comment by giogonzo** [comment]
ups sorry, I missed the approval.
It means we decided to avoid using skinnable here, and to investigate further in the longer term. Is there an issue for this? Can you please open one if not, @francescogior ?

Also, whatever it is, it could break every other component using skinnable in obscure ways... Should we remove it everywhere in the meantime @FrancescoCioria ?

**Pull request merged**